nextflow.config

```diff
@@ -126,6 +126,7 @@ params {
     cut_dada_ref_taxonomy    = false
     dada_addspecies_allowmultiple = false
     dada_taxonomy_rc         = false
+    dada_assign_chunksize    = 10000
```

Comment:
Isn't that a bit low? If one wants to keep the original behaviour one could set this to something ridiculously high, e.g. 1000000.

Reply:
I actually wanted specifically to have a realistic value so that chunking goes into effect when it might be beneficial (I didnt test when it is, might depend whether the compute resource is large enough to run all chunks in parallel). Do you suggest its better to keep the original behavior by choosing a high default? I am unsure whats best here.

Reply:
I have no strong opinion, and maybe you're right. We've had a couple of complaints about this, so maybe keep as it is now.